src/pages/library/dataBrowser-utils.js

```diff
@@ -116,10 +129,7 @@ export const useDataCatalog = () => {
     Utils.withBusyState(setLoading)
   )(async () => {
     const datasets = await Ajax(signal).Catalog.getDatasets()
```

Comment:
```suggestion
    const { result: datasets } = await Ajax(signal).Catalog.getDatasets()
```

Nit: based on the names, I'd expect `datasets` to be a list and `filterAndNormalizeDatasets` to take a list and return that list filtered and normalized. But datasets is actually an object `{ result: [...] }`.